### PR TITLE
Refactoring: LobbyTimer, Small typo of equipment interval debug msg

### PR DESCRIPTION
**Pull request by RedstoneFuture**

### CODE REVIEW

File: missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/timer/LobbyTimer.java [hunk 1]
```diff
@@ -47,14 +47,11 @@ public LobbyTimer(Game game, int startTime) {
...
     @Override
     public void tick() {
-        if (getGame().getPlayers().values().size() == 0) return;
+        if (getGame().getPlayers().size() == 0) return;
 
         for (MWPlayer mp : getGame().getPlayers().values()) {
-            if (mp.getPlayer() != null) {
-                mp.getPlayer().setLevel(seconds);
-            } else {
-                getGame().removePlayer(mp);
```
Comment (Butzlabben) [inline missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/timer/LobbyTimer.java:56]: Do we really no longer need this line? This was added as a kind of safeguard, but I don't know if this is needed any longer

Comment (daniel-naegele) [inline missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/timer/LobbyTimer.java:56]: Do we really no longer need this line? This was added as a kind of safeguard, but I don't know if this is needed any longer

Comment (RedstoneFuture) [reply missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/timer/LobbyTimer.java:56]: If the gameLeaveEvent from the EventHandler down't work correctly, it's not the solution for all problems, I think.

Comment (RedstoneFuture) [reply missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/timer/LobbyTimer.java:56]: Too many safeguards have side effects and in the end you don't understand what the plugin has done.

**Pull request merged**